### PR TITLE
VR-5480 Add docstrings for RegisteredModel and RegisteredModelVersion public methods

### PR DESCRIPTION
**Pull request by nhatsmrt**
WIP.

### CODE REVIEW

File: client/verta/verta/_registry/modelversion.py [hunk 1]
```diff
@@ -184,6 +201,23 @@ def log_artifact(self, key, asset, overwrite=False):
...
+
+        Parameters
+        ----------
+        key : str
+            Name of the artifact.
+
+        Returns
+        -------
+        str or bytes
+            Filesystem path or bytes representing the artifact.
```
Comment (convoliution) [inline client/verta/verta/_registry/modelversion.py]: This method can return the object itself, too!
(suggestion taken from `ExperimentRun.get_artifact()`)
```suggestion
            Filesystem path of the artifact, the artifact object, or a bytestream representing the
            artifact.
```

File: client/verta/verta/_registry/model.py [hunk 2]
```diff
@@ -122,6 +122,15 @@ def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=Non
         return registered_model
 
     def add_labels(self, labels):
+        """
+        Adds multiple labels to this Registered Model.
+
+        Parameters
+        ----------
+        labels : list of str
+            labels to add.
```
Comment (convoliution) [inline client/verta/verta/_registry/model.py]: Capitalize the descriptions of `Parameters`/`Returns`.
```suggestion
            Labels to add.
```

**Review by convoliution — CHANGES_REQUESTED** [review]

**Comment by nhatsmrt** [comment]
I think that's all?

**Review by convoliution — APPROVED** [review]
Nice, thanks!

**Pull request merged**